sites/skeleton.dev/src/routes/(inner)/docs/purgecss/+page.svelte

```diff
@@ -0,0 +1,131 @@
+<script lang="ts">
+	import LayoutPage from '$lib/layouts/LayoutPage/LayoutPage.svelte';
+	import { CodeBlock } from '@skeletonlabs/skeleton';
+</script>
+
+<LayoutPage>
+	<!-- Header -->
+	<header class="space-y-4">
+		<h1 class="h1">PurgeCSS</h1>
+		<!-- prettier-ignore -->
+		<div>
+			<div class="!flex flex-col md:flex-row justify-between items-start space-y-4 md:space-y-0 md:space-x-4">
+                <!-- prettier-ignore -->
+				<p>
+                    This guide covers <code class="code">vite-plugin-tailwind-purgecss</code>, a simple to use Vite plugin that thoroughly purges excess CSS via <a class="anchor" href="https://purgecss.com/" target="_blank">PurgeCSS</a>. While optional, this is highly recommended if you wish to minimize your production CSS bundle size.
+                </p>
+				<a class="btn variant-filled-secondary" href="https://github.com/AdrianGonz97/vite-plugin-tailwind-purgecss" target="_blank">
+					<i class="fa-brands fa-github" />
+					<span>View on GitHub</span>
+				</a>
+			</div>
+		</div>
+	</header>
+
+	<hr />
+
+	<section class="space-y-4">
+		<h2 class="h2">Introduction</h2>
+		<h3 class="h3">Motivation</h3>
+		<p>
+			Tailwind UI component libraries like Skeleton, Tailwind, and Flowbite provide a number of benefits, but come with an important caveat
+			- Tailwind generates classes for all imported components, regardless if they are used in the project or not. This leads to a larger
+			than necessary CSS bundle.
```

Comment:
```suggestion
			Tailwind UI component libraries like Skeleton and Flowbite provide a number of benefits, but come with an important caveat
			- Tailwind generates classes for all imported components, regardless if they are used in the project or not. This leads to a larger
			than necessary CSS bundle.
```
Tailwind doesn't really fit in this list. This is mainly for component libraries that are built with Tailwind.

Reply:
I meant to put Skeleton, Flowbite, and Daisy UI. I'll commit a fix shortly.